app/inbound/organisation/schema.js

```diff
@@ -0,0 +1,68 @@
+const Joi = require('joi')
+const { CALCULATION, ORGANISATION } = require('../../constants/types')
+
+const frnMin = 1000000000
+const frnMax = 9999999999
+const sbiMin = 105000000
+const sbiMax = 999999999
+const postcodeMax = 8
+
+module.exports = Joi.object({
+  sbi: Joi.number().integer().min(sbiMin).max(sbiMax).required().messages({
+    'number.base': 'sbi should be a type of number',
+    'number.integer': 'sbi must be an integer',
+    'number.min': `sbi should have a minimum value of ${sbiMin}`,
+    'number.max': `sbi should have a maximum value of ${sbiMax}`,
+    'any.required': 'The field sbi is not present but it is required'
+  }),
+  addressLine1: Joi.string().required().messages({
+    'string.base': 'addressLine1 should be a type of string',
+    'any.required': 'The field addressLine1 is not present but it is required'
+  }),
+  addressLine2: Joi.string().required().messages({
+    'string.base': 'addressLine2 should be a type of string',
+    'any.required': 'The field addressLine2 is not present but it is required'
+  }),
+  addressLine3: Joi.string().required().messages({
+    'string.base': 'addressLine3 should be a type of string',
+    'any.required': 'The field addressLine3 is not present but it is required'
+  }),
+  city: Joi.string().required().messages({
+    'string.base': 'city should be a type of string',
+    'any.required': 'The field city is not present but it is required'
+  }),
+  county: Joi.string().required().messages({
+    'string.base': 'county should be a type of string',
+    'any.required': 'The field county is not present but it is required'
+  }),
+  emailAddress: Joi.string().email().required().messages({
+    'string.base': 'emailAddress should be a type of string',
+    'string.email': 'emailAddress must be a valid email',
+    'any.required': 'The field emailAddress is not present but it is required'
+  }),
+  frn: Joi.number().integer().min(frnMin).max(frnMax).required().messages({
+    'number.base': 'frn should be a type of number',
+    'number.integer': 'frn must be an integer',
+    'number.min': `frn should have a minimum value of ${frnMin}`,
+    'number.max': `frn should have a maximum value of ${frnMax}`,
+    'any.required': 'The field frn is not present but it is required'
+  }),
+  name: Joi.string().required().messages({
+    'string.base': 'name should be a type of string',
+    'any.required': 'The field name is not present but it is required'
+  }),
+  postcode: Joi.string().max(postcodeMax).required().messages({
+    'string.base': 'postcode should be a type of string',
+    'string.length': `postcode should have a length of ${postcodeMax}`,
```

Comment:
you are using this is a max but then telling them it must have a length of exactly the post code max. so either you need to introduce a minimum for the same value to ensure it is that length, or you need to adjust the error message to say something like 'postcode should have a maximum length of...'